main.rb

```diff
@@ -1,51 +1,15 @@
 require_relative './app'
-
-def menu
-  puts 'Welcome to School Library app'
-  puts 'Select an option by entering a number: '
-  puts '1) List all books'
-  puts '2) List all people'
-  puts '3) Create a person'
-  puts '4) Create a book'
-  puts '5) Create a rental'
-  puts '6) List all rentals for a given person id'
-  puts '7) Exit'
-end
+require_relative './menu'
+require_relative './menu_executer'
 
```

Comment:
- [ ] @degisew and @Ceemos96 great work refactoring your app, however, you forgot to refactor your creation methods, kindly move methods that request input and input validation to the main.rb file or files whose responsibilities are to request or validate input only.
tip `you can create a create person method, that calls on private methods for create teacher and student so that you achieve the Single-responsibility Principle`

Reply:
Okay, Thanks! We will walk through it.